### PR TITLE
feat: updates Breadcrumb design and UX.

### PR DESCRIPTION
**Pull request by aditya-kumawat**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/atoms/breadcrumbs/__stories__/variants/lessThan4.story.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { Breadcrumbs } from '@/index';
+import { action } from '@storybook/addon-actions';
+
+// CSF format story
+export const lessThan4 = () => {
```
Comment (satyamyadav) [inline core/components/atoms/breadcrumbs/__stories__/variants/lessThan4.story.tsx]: This should be `lessThan4Levels` instead.

File: core/components/atoms/breadcrumbs/__stories__/variants/moreThan4.story.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { Breadcrumbs } from '@/index';
+import { action } from '@storybook/addon-actions';
+
+// CSF format story
+export const moreThan4 = () => {
```
Comment (satyamyadav) [inline core/components/atoms/breadcrumbs/__stories__/variants/moreThan4.story.tsx]: The story title (component name) should be `moreThan4Levels`.

**Review by satyamyadav — CHANGES_REQUESTED** [review]
@aditya-kumawat  Please add custom code block for docs page.

**Comment by aditya-kumawat** [comment]
> @aditya-kumawat Please add custom code block for docs page.

Let me update.

**Review by satyamyadav — APPROVED** [review]

**Pull request merged**